var/spack/repos/builtin/packages/papi/package.py

```diff
@@ -37,6 +38,7 @@ class Papi(Package):
     variant('powercap', default=False, description='Enable powercap interface support')
     variant('rapl', default=False, description='Enable RAPL support')
     variant('lmsensors', default=False, description='Enable lm_sensors support')
+    variant('sde', default=False, description='Enable software defined events')
```

Comment:
This variant doesn't seem to be used anywhere...

Reply:
The list of variants are passed to the install at papi "component" names on line 69:
`            configure_args.append(
                '--with-components={0}'.format(' '.join(
                    filter(lambda x: spec.variants[x].value, spec.variants))))`

Reply:
Magic!